frontend/packages/otr/src/interfaces/clerkInterpreter.ts

```diff
@@ -33,3 +33,9 @@ export interface ClerkInterpreterState {
   interpreters: Array<ClerkInterpreter>;
   status: APIResponseStatus;
 }
+
+export interface ClerkInterpreterOverviewState {
+  interpreter?: ClerkInterpreter;
+  status: APIResponseStatus;
+  interpreterDetailsStatus: APIResponseStatus;
```

Comment:
Mihin näitä eri `status`-kenttiä tarvitaan? `interpreterDetailsStatus` arvoa ei taideta ainakaan vielä muuttaa missään, onko se lainkaan tarpeen? Jos kenttää tarvitaan, niin sitten näiden eri kenttien nimet voisivat olla kuvaavampia.

Reply:
Näitä tosiaan käytettään kun editoidaan perustietoja / kielipareja / koulutuksia. Tämän ControlButtons olisi voinut jättää siihen tulevaan PR:ään mutta mietitään paremmat nimet näille editoinnin PR:ssä

Reply:
Sopii.